x-pack/plugins/maps/public/layers/sources/es_source/es_source.d.ts

```diff
@@ -20,6 +20,7 @@ export interface IESSource extends IVectorSource {
     limit: number,
     initialSearchContext?: object
   ): Promise<SearchSource>;
+  loadStylePropsMeta(args: unknown): Promise<unknown>; // todo
```

Comment:
args are 
```
layerName: string, 
style: VectorStyle, 
dynamicStyleProps: IDynamicStyleProperty[], 
registerCancelCallback: (requestToken: symbol, callback: () => void): void, 
nextMeta: VectorSourceRequestMeta
```

return type is 

```
RangeFieldMeta | CategoryFieldMeta
```

Reply:
thanks for the signature. the return type is actually a raw ES response. Will leave typing of that for another PR, since it bloats scope.